include/r502_types.h

```diff
@@ -154,6 +156,35 @@ typedef struct Command {
             uint8_t page_num;
         } read_notepad;
 
+        struct AuraLedConfig {
+            /*
+                Control code:
+                0x01 - breathing light
+                0x02 - flashing light
+                0x03 - light always on
+                0x04 - light always off
+                0x05 - light gradually on
+                0x06 - light gradually off
+            */
+            uint8_t ctrl;
+            /* Speed: 0x00-0xff, 256 gears, minimum 5s cycle */
```

Comment:
Newline between the member above and the description of the current one. Applicable for the rest of the members in this `struct`.